tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -2587,7 +2587,14 @@ void MklLayoutRewritePass::CopyAttrsConvCheckConstFilter(const Node* orig_node,
   // Check and set filter attribute.
```

Comment:
Please add a comment saying that the attribute is only set when the original node has this attribute.